### PR TITLE
Recompile less often.

### PR DESCRIPTION
**Pull request by ltratt**
Using `rerun_except` helps avoid unnecessary rebuilds just because we've changed a language test.

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -7,20 +7,20 @@
...
 fn main() -> Result<(), Box<std::error::Error>> {
+    rerun_except(&["/lang_tests/*.som"])?;
+
     let lex_rule_ids_map = CTParserBuilder::new()
         .yacckind(YaccKind::Grmtools)
         .process_file_in_src("lib/compiler/som.y")?;
     LexerBuilder::new()
         .rule_ids_map(lex_rule_ids_map)
         .process_file_in_src("lib/compiler/som.l")?;
+
```
Comment (vext01) [inline build.rs]: This should have been in a separate commit really? I don't fell super-strongly.

**Comment by vext01** [comment]
This is fine. Up to you if you want to put that blank line in a separate commit.

**Comment by ltratt** [comment]
I think the blank line is not too offensive: I added it purely because the method had grown a little longer.

**Comment by vext01** [comment]
OK, ready to merge, or are you squashing?

**Comment by ltratt** [comment]
I think this is ready as-is.

**Pull request merged**